go.mod

```diff
@@ -699,12 +700,15 @@ require (
 	go.opentelemetry.io/collector/processor v0.97.0
 	go4.org/intern v0.0.0-20230525184215-6c62f75575cb
 	gotest.tools v2.2.0+incompatible
+	k8s.io/cli-runtime v0.29.0
+	k8s.io/kubectl v0.29.0
```

Comment:
Something I did not think about during previous reviews (sorry), did you check binary size?

Reply:
👋🏻 Just checked the binary size and compared between `main` and my branch: both produce a `cluster-agent` binary of `160MB` 🎉